aidial_adapter_bedrock/llm/model/claude/v3/converters.py

```diff
@@ -101,7 +103,7 @@ async def _to_claude_image(
     raise ValidationError("Attachment data or URL is required")
 
 
-async def _to_claude_content(
+async def _basic_message_to_claude_content(
```

Comment:
**content** in `_to_claude_content` doesn't tell much. I suggest more light-weight naming like

```
_to_claude_message
_to_claude_tool_call
_to_claude_tool_result
```

Reply:
Fixed